### DISCUSSION TITLE
Customize Translation task

### DISCUSSION THREAD

**Discussion by Bodawen** [openai/whisper#1167] [General]
Thanks for the great job!

I am interested in the translation, currently whisper only supports the X to English translation. But it will be great that user can translate English to a special language. So can you give me some details how the translation works? Is that possible to build the English to X translation task by myself?

**Comment by jongwook** [comment]
The model is not trained for that; it occasionally does it when you set `task="translate"` and `language` as the target language, but it's mostly due to imperfect language id in the training data, and it won't be very reliable.

You could try fine-tuning the model on this task, but it will require a nontrivial amount of training data for translation.

**Comment by Francoyy** [comment]
I would like to second that. It is a very needed feature outside the US. The translation should not even need the video file, just the .srt file + the full text based script to translate.
